maven-plugin-plugin/src/it/help-basic-deprecated-annotation-only/expected-help.txt

```diff
@@ -0,0 +1,25 @@
+[INFO] help 1.0-SNAPSHOT
+  Tests generation and compilation of the help mojo.
+
+help:test
+  Deprecated. No reason given
+
+  MOJO-DESCRIPTION. Some 'quotation' marks and backslashes '\\', some important
+  javadoc
+  and an inline link to AnotherMojo.
+
+  Available parameters:
+
+    defaultParam (Default: escape\backslash)
+      This parameter uses 'quotation' marks and backslashes '\\' in its
+      description. Those characters must be escaped in Java string literals.
+
+    deprecatedParam
+      Deprecated. No reason given
+
+      This parameter is deprecated.
```

Comment:
This is weird. I see basically the same information twice, no?

Reply:
it is ok, 
 - first line because parameters has deprecated annotation
 - second line from parameter javadoc

Reply:
Can't it be reasonably combined into: This parameter is deprecated. No reason given.??

Reply:
No
as I said there are two different source of it 
- one deprecated annotation - annotation does not have a message so default one is used "No reason given" the same will be if we provide javadoc tag without message

- second line is from javadoc about parameters - we can change this text in test

Reply:
I understand that, but at the end for the warning/help this makes not difference. It is deprecated after all. Since you know that `@Deprecated` is the same as `@deprecated`, I am still convinced that it can be reasonably merged. It can't deprecate it twice anyway.

Reply:
I changed test a little - look now